update/Makefile

```diff
@@ -23,14 +23,16 @@ LD_FLAGS = -ldflags " \
 			-X 'github.com/percona/pmm/version.FullCommit=$(PMM_RELEASE_FULLCOMMIT)' \
 			-X 'github.com/percona/pmm/version.Branch=$(PMM_RELEASE_BRANCH)' \
 			"
+prep: 													## OL9-specific
```

Comment:
Can you please add better comment and align it. We use them for `make help` output generation.

Reply:
BTW why it's needed for `9` but not needed for `7`? What is the difference?

Reply:
Ok, will do. RHEL 7 and 9 use different versions of git, so it's mainly because git was erroring out.

Reply:
Commented.